akka-distributed-data/src/multi-jvm/scala/akka/cluster/ddata/ReplicatorMapDeltaSpec.scala

```diff
@@ -97,24 +129,44 @@ object ReplicatorMapDeltaSpec extends MultiNodeConfig {
         availableForRemove.toVector(rnd.nextInt(availableForRemove.size))
     }
 
-    (0 to (30 + rnd.nextInt(10))).map { _ ⇒
-      rnd.nextInt(4) match {
+    (0 to (120 + rnd.nextInt(10))).map { _ ⇒
```

Comment:
I'm not sure it is better coverage just by increasing this, e.g. higher chance that a Set will just be filled with all values if removals are not used. 50?